pkg/kubectl/cmd/cmd.go

```diff
@@ -252,6 +255,12 @@ var (
 
 // NewKubectlCommand creates the `kubectl` command and its nested children.
 func NewKubectlCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cobra.Command {
+	if f := os.Getenv("KUBECTL_FEATURE_GATES"); f != "" {
+		if e := utilfeature.DefaultFeatureGate.Set(f); e != nil {
```

Comment:
Normally it would, but I want to print to the `io.Writer` with the same name from the enclosing scope.